contributing.md

```diff
@@ -112,6 +112,25 @@ tests/test_default.yml`
    there are any errors. If there is no problem with your contribution, the mantainer
    will merge it to the main project.
 
+### Find other images for testing
+To use the CentOS 6, CentOS 7, CentOS 8 images, please checkout the link below:<br/>
+https://cloud.centos.org/centos/6/images/CentOS-6-x86_64-GenericCloud-1907.qcow2c<br/>
+https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-2003.qcow2c<br/>
+https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2
+
+- `/6/images/` stands for CentOS 6 images, we can also change it to `/7/images/` to use
+CentOS 7 images, change it to `/8/images/` to use CentOS 8 images.
+- For qemu testing cases, we prefer the image architecture to be `x86_64-GenericCloud`.
+- `2003` in `CentOS-7-x86_64-GenericCloud-2003.qcow2c` stands for image released in
+March 2020.
+- We can use the image with extension `.qcow2` and `.qcow2c`.
+- To save the image, right click on the link above, then select "Save link as...".
+
+For Fedora, we recommend to use the latest qcow2 images from<br/>
+http://mirrors.kernel.org/fedora/releases/31/Cloud/x86_64/images/Fedora-Cloud-Base-31-1.9.x86_64.qcow2<br/>
+http://mirrors.kernel.org/fedora/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2
```

Comment:
The Fedora 31 one is almost a year old, so "latest" is an overstatement here. Could one use images from https://kojipkgs.fedoraproject.org/compose/cloud/ ?

Reply:
I compared the image download from http://mirrors.kernel.org/fedora/releases/31/Cloud/x86_64/images/Fedora-Cloud-Base-31-1.9.x86_64.qcow2, http://mirrors.kernel.org/fedora/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2, and https://kojipkgs.fedoraproject.org/compose/cloud/ while testing on some playbooks, they all failed. Can anyone nominate a testing playbooks working on Fedora ?

Reply:
works for me
```
curl -o /path/to/Fedora-Cloud-Base-31-20200929.0.x86_64.qcow2 https://kojipkgs.fedoraproject.org/compose/cloud/latest-Fedora-Cloud-31/compose/Cloud/x86_64/images/Fedora-Cloud-Base-31-20200929.0.x86_64.qcow2
cd network/tests
ANSIBLE_STDOUT_CALLBACK=debug TEST_SUBJECTS=/path/to/Fedora-Cloud-Base-31-20200929.0.x86_64.qcow2 ansible-playbook -vv -i /usr/share/ansible/inventory/standard-inventory-qcow2 tests_ethtool_features_nm.yml
....
PLAY RECAP *********************************************************************
/path/to/Fedora-Cloud-Base-31-20200929.0.x86_64.qcow2 : ok=101  changed=26   unreachable=0    failed=0    skipped=39   rescued=1    ignored=0
```

Reply:
@pcahyna  the image from https://kojipkgs.fedoraproject.org/compose/cloud/  also works

Reply:
@liangwen12year IMO, we should use the image from https://kojipkgs.fedoraproject.org/compose/cloud/

Reply:
@ffmancera updated the commit